package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "onboarding_backend",
+  "version": "1.0.0",
+  "description": "## Project's Description This repo contains all files for my journey during the back-end oboarding at Instituto Taqtile.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "graphql": "^16.8.1"
```

```suggestion
    "graphql": "16.8.1"
```

Lembra de sempre que instalar um novo pacote lockar a versão dele

---

pq é importante lockarmos versão?

---

É para garantir que não vai existir inconsistência nos resultados por conta de versionamento.